app/views/layouts/application.html.erb

```diff
@@ -4,15 +4,15 @@
     <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track" => true %>
     <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
     <%= csrf_meta_tags %>
-    <title>Hogwarts
+    <title>Hogwarts</title>
   </head>
   <body>
     <nav>
       <h1>Welcome to Hogwarts!</h1>
-      <h2><%= link_to "View Houses", houses_path %> | <%= link_to "View Students", student_path %> | <%= link_to "Enroll", student_path %></h2>
+      <h2><%= link_to "View Houses", houses_path %> | <%= link_to "View Students", students_path(@student) %> | <%= link_to "Enroll", new_student_path %></h2>
```

Comment:
`students_path` takes the user to the index page for students. Since there's no need to identify a particular student, you don't need to pass in `@student` as an argument to the path helper. Just `students_path` will work.